packages/keystone/bin/utils.js

```diff
@@ -0,0 +1,37 @@
+const keystone = require('@keystone-alpha/core');
+const endent = require('endent');
+const path = require('path');
+
+function getEntryFileFullPath(args, { exeName, _cwd }) {
+  const entryFile = args['--entry'] ? args['--entry'] : keystone.DEFAULT_ENTRY;
+  try {
+    return Promise.resolve(require.resolve(path.resolve(_cwd, entryFile)));
+  } catch (error) {
+    return Promise.reject(
+      new Error(endent`
+        --entry=${entryFile} was passed to ${exeName}, but '${entryFile}' couldn't be found in ${process.cwd()}.
+        Ensure you're running ${exeName} from within the root directory of the project.
+      `)
+    );
+  }
+}
+
+function executeDefaultServer(args, entryFile) {
+  const port = args['--port'] ? args['--port'] : keystone.DEFAULT_PORT;
+
+  return keystone
+    .prepare({ entryFile, port })
```

Comment:
`keystone.prepare()` needs to accept a new parameter `adminDistDir` so it knows where to load the `adminUI` build files from.

Then later within `packages/admin-ui/server/AdminUI.js`, `'dist'` can be set to the value of `adminDistDir` (see comment below).